### PR TITLE
chore(prepare next release): Preparing release 8.6.0

### PR DESCRIPTION
**Pull request by robertohuertasm**
Waiting for the approval of the pending PRs before merging and publishing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,43 @@
...
+- Feature: Support for `Grunt` folder. ([@KingDarBoja](https://github.com/KingDarBoja) in [#1988](https://github.com/vscode-icons/vscode-icons/pull/1988))
+- Feature: Support for `windows, linux & macos` folders. ([@KingDarBoja](https://github.com/KingDarBoja) in [#1986](https://github.com/vscode-icons/vscode-icons/pull/1986))
+- Feature: Support for `modules` folder. ([@KingDarBoja](https://github.com/KingDarBoja) in [#1985](https://github.com/vscode-icons/vscode-icons/pull/1985))
+- Feature: Support for `Sentry`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#1984](https://github.com/vscode-icons/vscode-icons/pull/1984))
+- Feature: Support for `JSS`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#1983](https://github.com/vscode-icons/vscode-icons/pull/1983))
+- Feature: Support for `Moleculer`. ([@icebob](https://github.com/icebob) in [#1980](https://github.com/vscode-icons/vscode-icons/pull/1980))
+- Feature: Support for `APL`. ([@KingDarBoja](https://github.com/KingDarBoja) in [#1978](https://github.com/vscode-icons/vscode-icons/pull/1978))
+
+
```
[the remark this answers is not in the document]
Comment (jens1o) [reply CHANGELOG.md]: I think this is a valid point. In earlier releases, we only had one trailing new-line.

[the remark this answers is not in the document]
Comment (robertohuertasm) [reply CHANGELOG.md]: Yes, I will fix it

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,42 @@
 # Change Log
 
+## 8.6.0 (April 722, 2019)
```
Comment (revam) [inline CHANGELOG.md]: @robertohuertasm I think you meant "April 22, 2019" and not "April 722, 2019".

Just updated the package to the latest release and spotted this error, so this is just a heads-up from a random user.

Comment (JimiC) [reply CHANGELOG.md]: We were in a hurry and when you are in a hurry :hankey: :hankey: happens.

Comment (robertohuertasm) [reply CHANGELOG.md]: Thanks for catching that!

Comment (robertohuertasm) [reply CHANGELOG.md]: Fixed

**Pull request merged**